tests/unit/test_fileio.py

```diff
@@ -37,7 +39,12 @@ def test_attributes(self):
         self.assertTrue(reader.seekable())
         self.assertTrue(reader.readable())
         self.assertFalse(reader.writable())
-        self.assertEqual(256, reader._chunk_size)
+        self.assertEqual(reader._chunk_size, 256)
+        self.assertEqual(reader._retry, DEFAULT_RETRY)
+
```

Comment:
I would generally split this into two test cases, e.g. `test_ctor_w_defaults` and `test_ctor_w_explicit`).